groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/YtterbiumChain.groovy

```diff
@@ -0,0 +1,28 @@
+import globals.Globals
+
+import static gregtech.api.unification.material.Materials.*;
+import gregtech.api.unification.material.MarkerMaterials;
+import static gregtech.api.unification.ore.OrePrefix.dye;
+
+BR = recipemap('batch_reactor')
+REACTION_FURNACE = recipemap('reaction_furnace')
+
+BR.recipeBuilder()
+        .fluidInputs(fluid('ytterbium_amalgam') * 1000)
+        .fluidInputs(fluid('hydrochloric_acid') * 5000)
+        .outputs(metaitem('dustYtterbiumChloride') * 4)
```

Comment:
Again, given the very large amount of water, ytterbium chloride would probably manage to dissolve in it (17 g/100 mL).